src/test/regress/output/multi_outer_join.source

```diff
@@ -53,6 +53,10 @@ SELECT master_create_distributed_table('multi_outer_join_third', 't_custkey', 'a
  
 (1 row)
 
+-- Manually colocate tables multi_outer_join_left, multi_outer_join_right and multi_outer_join_third
```

Comment:
I think this is ugly. Why not directly call `create_distributed_table()` and get rid of the old APIs?